File: C++/1274.cpp
#include <iostream>

using namespace std;

int main(){
	int n, cnt = 0;
	cin >> n;
	
	for(int i = 1; i <= n; i++){
		if(n%i==0) cnt += 1;
	}
	
	if(cnt==2) cout << "prime";
	else cout << "not prime";
}

File: Java/1274.java
import java.util.Scanner;

public class Main{
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int n = sc.nextInt();
		int cnt = 0;
		
		for(int i = 1; i <= n; i++) {
			if(n%i==0) cnt++;
		}
		
		if(cnt==2) System.out.println("prime");
		else System.out.println("not prime");
	}
}